pkg/app/web/tsconfig.json

```diff
@@ -5,7 +5,7 @@
     },
     "allowUnreachableCode": false,
     "baseUrl": ".",
-    "module": "CommonJS",
+    "module": "esnext",
```

Comment:
nits, is it safe to use `esnext` over a specific version of js 👀

Reply:
@khanhtc1202 Yes, it's safe because we use `webpack` after compiling with `typescript`.
If it is broken, the webpack build should fail.

Reply:
got it, thanks for your explaining 🙆‍♀️